large-file-uploader-app/assets/iframe.html

```diff
@@ -21,7 +21,7 @@
 					function(data) {
 						var accountEntryId = data['ticket.customField:custom_field_360013377572']
 
-						if( accountEntryId !== null && accountEntryId !== '' ) {
+						if( accountEntryId && accountEntryId !== '' ) {
```

Comment:
Minor detail, can you remove the space inside the `()` and add a space after the `if`?
For example: 
`if (accountEntryId && accountEntryId !== '') {` instead of `if( accountEntryId && accountEntryId !== '' ) {`

Reply:
Of course! It looks standardjs `--fix` command made me a little lazy these days :)